content/notebooks/climate_indicators/PAVICStutorial_ClimateDataAnalysis-3Climate-Indicators.ipynb

```diff
@@ -767,31 +767,37 @@
     }
    ],
    "source": [
-    "from siphon.catalog import TDSCatalog\n",
-    "from xclim import atmos\n",
-    "from clisops.core import subset\n",
-    "import xarray as xr\n",
-    "import geopandas as gpd\n",
```

Comment:
@Zeitsperre This import re-ordering causes Jenkins failure because extra warnings are printed out ! See http://jenkins.ouranos.ca/job/PAVICS-e2e-workflow-tests/job/master/1420/console
![Screenshot from 2022-01-18 21-00-47](https://user-images.githubusercontent.com/11966697/150049793-ce78c64a-85b2-4de0-bd22-c3bc77675117.png)

I can commit a new version to undo this import ordering but it is possible to tell the pre-commit formatter to not touch import ordering?  Else the problem will comes back again.